warn/warn_cosmetic.go

```diff
@@ -124,80 +124,104 @@ func loadOnTopWarning(f *build.File, fix bool) []*Finding {
 	return findings
 }
 
-func outOfOrderLoadWarning(f *build.File, fix bool) []*Finding {
-	findings := []*Finding{}
+// compareLoadLabels compares two module names
+func compareLoadLabels(load1Label, load2Label string) bool {
```

Comment:
Add examples/explanation in a comment to show how it differs from alphanumeric sort (is it just about the prefix "@"?)

Reply:
Done.